### PR TITLE
BIT-192 wallet to ansible vault

### PR DESCRIPTION
**Pull request by isabella618033**

### CODE REVIEW

File: tests/unit_tests/bittensor_tests/test_axon.py [hunk 1]
```diff
@@ -11,18 +11,24 @@
...
     nounce = datetime.now().strftime(format= '%m%d%Y%H%M%S%f')
     message  = nounce+str(wallet.hotkey.ss58_address) 
     spliter = 'bitxx'
-    signature = spliter.join([nounce,str(wallet.hotkey.ss58_address),wallet.hotkey.sign(message)])
-    return signature
-
 
-wallet.create_new_coldkey( use_password=False, overwrite = True)
-wallet.create_new_hotkey( use_password=False, overwrite = True)
+    with mock.patch('getpass.getpass', return_value = ''):
```
Comment (unconst) [inline tests/unit_tests/bittensor_tests/test_axon.py]: The axon should never ask password in the tests.

File: bittensor/utils/wallet_utils.py [hunk 2]
```diff
@@ -0,0 +1,268 @@
+""" Functions for wallets, including key extraction, storage, and user interface,
+other then
```
Comment (unconst) [inline bittensor/utils/wallet_utils.py]: incomplete sentence

File: bittensor/utils/wallet_utils.py [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+class CryptoKeyError(Exception):
+    """ Exception for invalid signature, key, token, password, etc 
+        Overwrite the built-in CryptoKeyError
+    """
+
+class KeyFileError(Exception):
+    """ Overwrite the built-in CryptoKeyError
+    """
+
+__SALT = b"Iguesscyborgslikemyselfhaveatendencytobeparanoidaboutourorigins"
```
Comment (unconst) [inline bittensor/utils/wallet_utils.py]: hahaha

File: bittensor/utils/wallet_utils.py [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+        if os.path.isfile(full_path):
+            if os.access(full_path, os.W_OK):
+                if overwrite:
+                    return full_path
+
+                elif _user_interface.may_overwrite( full_path ):
+                    return full_path
+
+                else:
+                    quit()
```
Comment (unconst) [inline bittensor/utils/wallet_utils.py]: is quit()

**Review by shibshib — APPROVED** [review]

File: bittensor/utils/wallet_utils.py (hunk 3 above)
Comment (isabella618033) [reply bittensor/utils/wallet_utils.py]: hahaha

Comment (shibshib) [reply bittensor/utils/wallet_utils.py]: hehehe

**Pull request merged**